recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: plasmapy
+  version: {{ version }}
+
+source:
+  fn: plasmapy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/p/plasmapy/plasmapy-{{ version }}.tar.gz
+  sha256: 5048e95700274836443b1f7e533622d7ea7f159b93e938991b7edb7fe43eccc6
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  skip: True    # [py < 36]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python 3.6.*
```

Comment:
Is it possible for these pins to float and be driven by the general set of pins for conda-forge.

In particular we are no longer setting our lower bound to versions of scipy and numpy that are this old.  Also python 3.7 is pretty solid by now

Reply:
It took me a while to understand what you mean (woo 6 AM), but I think I get it now. Is commit 187db7cef042c18add25f5fb59284e8792f3689a all right?